Resources/Lang/en_US.json

```diff
@@ -2709,6 +2713,9 @@
   "SoulCollectorTitle": "SOUL COLLECTOR",
   "CollectOwnSoulOpt": "Can collect their own soul",
   "SoulCollectorSelfVote": "Host settings do not allow you to collect your own soul",
+  "SoulCollectorToDeath": "You have become Death!!!",
+  "SoulCollectorTransform": "<font=\"Brook SDF\"><align=\"center\"><size=150%>Now <b><color=#ff174f>Soul Collector</b></color> has become <b><color=#ff174f>Death</b></color>, Destroyer of Worlds and Horseman of the Apocalypse!</font><font=\"VCR SDF\"><br><br><size=115%><u>Find them and vote them out before it's <color=#ff174f>too late</color>!</u></size></align></font>",
```

Comment:
the slashes break these string, it's normally required in C sharp but not here. 
"<font="Brook SDF"><align="center"><size=150%>Now <b><color=#ff174f>Soul Collector</b></color> has become <b><color=#ff174f>Death</b></color>, Destroyer of Worlds and Horseman of the Apocalypse!</font><font="VCR SDF"><br><br><size=115%><u>Find them and vote them out\n<voffset=-0.5em>before it's <color=#ff174f>too late</color>!</voffset></u></size></align></font>" is fix I added a voffset because off this: 
![image](https://github.com/0xDrMoe/TownofHost-Enhanced/assets/143632280/e57f0032-1e6e-4685-ac08-43b7d2b38427)
You can remove it if you don't want.